src/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/Mocks/ProjectValueDataSource.cs

```diff
@@ -58,5 +53,15 @@ protected override void Initialize()
             // block has finished.
             await Task.WhenAll(_broadcastBlock.Completion, targetBlock.Completion);
         }
+
+        public async Task SendAsync(T value)
+        {
+            EnsureInitialized(true);
+
+            _version++;
```

Comment:
Is this going to be called from multiple threads?

Reply:
No, this is test code.